src/components/UI/Overlay/OverlayContent/DeleteResolverConfirmation.tsx

```diff
@@ -1,7 +1,6 @@
 import React, { FunctionComponent } from "react";
 import { useOverlayContext } from "../../../../common/context/OverlayContext";
```

Comment:
able to make delete resolver work in storybook demo? that will mean need to wrap overlay context provider in order for the story to work. can refer to `Overlay` storybook implementation